src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/AmendApplicationAboutToSubmitHandlerTest.java

```diff
@@ -225,6 +230,100 @@ public void givenCase_whenApplicantNotRepresentedBySolicitor_thenShouldDeleteApp
         assertNull(responseData.getContactDetailsWrapper().getSolicitorPhone());
     }
 
+    @Test
+    public void givenNullPostCode_whenApplicantAndRespondentNotRepresentedBySolicitor_thenHandlerThrowError() {
+        // Arrange
+        FinremCallbackRequest finremCallbackRequest = buildCallbackRequest();
+        FinremCaseData finremCaseData = finremCallbackRequest.getCaseDetails().getData();
+        Address address = new Address();
+        address.setCounty("West Yorkshire");
+        finremCaseData.getContactDetailsWrapper().setApplicantAddress(address);
+        finremCaseData.getContactDetailsWrapper().setRespondentAddress(address);
+        finremCaseData.getContactDetailsWrapper().setApplicantRepresented(YesOrNo.NO);
+        finremCaseData.getContactDetailsWrapper().setConsentedRespondentRepresented(YesOrNo.NO);
+
+        GenericAboutToStartOrSubmitCallbackResponse<FinremCaseData> response = handler.handle(finremCallbackRequest, AUTH_TOKEN);
+
+        assertEquals(2, response.getErrors().size());
+        assertTrue(response.getErrors().contains("Postcode field is required for applicant address."));
```

Comment:
Minor thing but you can use 
```
assertThat(response.getErrors(), containsInAnyOrder("Postcode field is required for applicant address.",
            "Postcode field is required for respondent address."));
```

Although I think in this case you can test the ordering explicitly so
`assertEquals("Postcode field is required for applicant address.", response.getErrors().get(0));`

Reply:
[f626845](https://github.com/hmcts/finrem-case-orchestration-service/pull/1753/commits/f626845fb39bbf77d699a1193d52ddb1341cd474)